.travis.yml

```diff
@@ -29,7 +29,9 @@ matrix:
     - php: 7.3
       env: WP_VERSION=5.2 WP_MULTISITE=1
     - php: 7.3
-      env: WP_VERSION=latest WP_MULTISITE=1
+      env:
+        - WP_VERSION=latest WP_MULTISITE=1
```

Comment:
```suggestion
        env: WP_VERSION=latest WP_MULTISITE=1 PHPCS=1
```

Reply:
@rfmeier Looks like the other two lines did not get removed. Let me see if I can get a multi-line suggestion working.